snapshots/NonceManagerTest.json

```diff
@@ -1,3 +1,3 @@
 {
-  "invalidateNonce": "45005"
```

Comment:
why the huge diff?

Reply:
mine actually has the same on main when i run it locally 🤔